selenium_tests/cross_listing/remove_cross_listing_tests.py

```diff
@@ -0,0 +1,56 @@
+from ddt import ddt, data, unpack
+
+from selenium_common.base_test_case import get_xl_data
+
+from selenium_tests.cross_listing.cross_listing_base_test_case import \
+    CrossListingBaseTestCase, TEST_DATA_CROSS_LISTING_MAPPINGS
+
+@ddt
+class RemoveCrossListingTests(CrossListingBaseTestCase):
+
+    @data(*get_xl_data(TEST_DATA_CROSS_LISTING_MAPPINGS))
+    @unpack
+    def test_remove_mapping(self, test_case_id, primary_cid,
+                            secondary_cid, expected_result,
+                            expected_alert_text):
+
+        """
+        Note: ICOMMONS_REST_API_HOST environment needs to match the LTI tool
```

Comment:
We can remove this comment - the backend isn't changing the cache at all for these. This was true for the people tests, but isn't relevant for these tests.